src/config.ts

```diff
@@ -313,3 +305,54 @@ export const SOFTWARE_VERSION = (() => {
     // into <pre> or <code> where it will create an unnecessary newline.
     return /^(.*)$/m.exec(versionFile)?.at(0) ?? defaultText;
 })();
+
+
+// Command line related functions
+
+/**
+ * Grabs an argument from the command line and checks if it exists.
+ * @param arg Argument name
+ * @returns True if the argument is present, otherwise false.
+ */
+function getCommandlineArgumentPresent(arg: string): boolean {
+    return process.argv.includes(arg);
+}
```

Comment:
It could be, but I'd like to at the least follow an authoritative source and use their terminology when referring to command line arguments. If you can find something more appropriate then please let me know. I am not familiar with interfacing with the OS or shell or wherever the arguments array actually comes from. 

If you think we should be even more specific by calling them `longOptionNames` then I'd agree.

Reply:
Honestly, I was thinking of renaming them to eg. `getNonPOSIXCommandLineStringLongOption`, just to be clear that it's not fully compliant (no `=`)

Reply:
(Tiny infodump: "argv" comes from the POSIX C argument name for the argument list, passed along to `execv`/`execve`)

Reply:
`String` in the name is redundant, we're not parsing anything. There's no need to say it's not posix compliant, but I am genuinely interested to know where this convention came from, if we don't know it shouldn't block this PR.

Reply:
I have no clue where the convention came from, nor am I able to find anything about it, but I know it's fairly common in shell scripts.

Reply:
The intent for explicitly saying `String` was to differentiate it from `Path`, by the way.

Reply:
As for the option vs long option thing, I don't think it really matters since the way it's implemented perfectly allows you to do eg. `-c /some/path` if you wanted short options, since the dashes are passed in the argument name.